### PR TITLE
Delete helm-2to3-migration app after migration

### PR DESCRIPTION
**Pull request by tomahawk28**
## Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Comment by tomahawk28** [comment]
I will update the changelog right before the merge (since collector changes are ready).

File: service/controller/app/resource/releasemigration/resource.go [hunk 1]
```diff
@@ -142,7 +105,7 @@ func (r *Resource) ensureReleasesMigrated(ctx context.Context, k8sClient k8sclie
 			}()
 
 			opts := helmclient.InstallOptions{
-				ReleaseName: "helm-2to3-migration",
+				ReleaseName: migrationApp,
```
Comment (tomahawk28) [inline service/controller/app/resource/releasemigration/resource.go:108]: Align with others.

File: service/controller/app/resource/releasemigration/resource.go [hunk 2]
```diff
@@ -67,49 +67,12 @@ func (r *Resource) Name() string {
...
-		charts[chart.Name] = true
-	}
-
-	lo := metav1.ListOptions{
-		LabelSelector: fmt.Sprintf("%s=%s", "OWNER", "TILLER"),
-	}
-
-	// Check whether helm 2 release configMaps still exist.
-	cms, err := k8sClient.K8sClient().CoreV1().ConfigMaps(tillerNamespace).List(lo)
+func (r *Resource) deleteMigrationApp(ctx context.Context, helmClient helmclient.Interface, tillerNamespace string) error {
```
Comment (tomahawk28) [inline service/controller/app/resource/releasemigration/resource.go:70]: This is actual changes, deleting the migration app here.

File: service/controller/app/resource/releasemigration/create.go [hunk 3]
```diff
@@ -151,6 +151,7 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 	// It means helm v3 release migration is finished.
 	if !hasConfigMap && hasSecret {
 		err = r.uncordonChart(ctx, cc.Clients.K8s.G8sClient())
+		err = r.deleteMigrationApp(ctx, cc.Clients.Helm, tillerNamespace)
```
Comment (rossf7) [inline service/controller/app/resource/releasemigration/create.go:154]: This looks a bit off and the go linter caught it which is cool.

Comment (tomahawk28) [reply service/controller/app/resource/releasemigration/create.go:157]: Thanks, I fixed this and added a changelog, PTAL :)

**Comment by tomahawk28** [comment]
going with a ping.

**Pull request merged**